docs/doc-writer.php

```diff
@@ -71,10 +71,10 @@ function get_replace_position($lines) {
 }
 
 function build_documentation($docs) {
-    usort(
-        $docs,
-        function($a, $b) { return strcmp($a->methodName, $b->methodName); }
-    );
+//    usort(
```

Comment:
Check. If you're happy with it now let's ship this then. Down the line we can always add an annotation for the doc ordering. It'd be sad if folks would shuffle around code in `Observable` just because it would suit the docs better.